components/ShowDatesList/index.js

```diff
@@ -0,0 +1,131 @@
+import moment from "moment";
+import { useState } from "react";
+import {
+  StyeldDatesList,
+  StyledDatesDetailCard,
+  StyledDatesDetailCardButton,
+  StyledDatesListRow,
+} from "./styled";
+
+export default function ShowDateList({ dates, places }) {
+  const sortDates = [...dates]
```

Comment:
`filter` already creates a copy of an array, so there is no need to "clone" the array first. We can just do. 

``` js
const sortDates = dates
  .filter(
    (date) =>
      moment(date.data.date).format(`YYYY-MM-DD`) >=
      moment().format(`YYYY-MM-DD`)
  )
  .sort((a, b) => new Date(a.data.date) - new Date(b.data.date));
```